Marlin/src/gcode/feature/pause/M600.cpp

```diff
@@ -43,73 +48,154 @@
  *  L[distance] - Extrude distance for insertion (positive value) (manual reload)
  *  B[count]    - Number of times to beep, -1 for indefinite (if equipped with a buzzer)
  *
+ * Spool_Swap : Tool change after filament runout
+ *
+ *  T[int]: 0=disable : 1/2/3 Enable the max number of swapping
+ *
  *  Default values are used for omitted arguments.
  *
  */
 void GcodeSuite::M600() {
-
-  #if ENABLED(HOME_BEFORE_FILAMENT_CHANGE)
+	
+  #if ENABLED(PAUSE_SPOOL_SWAP) 
+    //extern bool swap_spool_enabled;
+	//extern int16_t swap_spool_stop;
+	const int swap_spool_tmp = parser.intval('T',(FIL_RUNOUT_SENSORS));             
+    
+	// M600 T0/T1/2/3/4 Enable/Disable     
+	if (swap_spool_tmp<FIL_RUNOUT_SENSORS){
+	  if (swap_spool_tmp>0){
+	    // Activate or deactivate if next extruder available
+	    swap_spool_enabled = true ;
+		//Apply max number of swapping 
+		swap_spool_stop = swap_spool_tmp;
+	  }
```

Comment:
here is the management of the M600 T arguments
Trust , or try to understand lol